diff --git a/node_modules/@whiskeysockets/baileys/lib/Utils/validate-connection.js b/node_modules/@whiskeysockets/baileys/lib/Utils/validate-connection.js
index ba7f9b4..0dca531 100644
--- a/node_modules/@whiskeysockets/baileys/lib/Utils/validate-connection.js
+++ b/node_modules/@whiskeysockets/baileys/lib/Utils/validate-connection.js
@@ -15,12 +15,12 @@ const getUserAgent = (config) => {
         },
         platform: proto.ClientPayload.UserAgent.Platform.WEB,
         releaseChannel: proto.ClientPayload.UserAgent.ReleaseChannel.RELEASE,
-        osVersion: '0.1',
+        osVersion: '10.0.22631',
         device: 'Desktop',
-        osBuildNumber: '0.1',
-        localeLanguageIso6391: 'en',
-        mnc: '000',
-        mcc: '000',
+        osBuildNumber: '22631.4890',
+        localeLanguageIso6391: 'es',
+        mnc: '310',
+        mcc: '722',
         localeCountryIso31661Alpha2: config.countryCode
     };
 };
